tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -13598,6 +13606,78 @@ INLINE void static add_property_struct(char *func_name, int nreductions,
 }
 #endif
 
+void 
+add_vscale() 
+{
+  char token[60];
+  sprintf(token, "%d,%d", flg.min, flg.max);
+  print_token(" \"vscale-range\"=\"");
+  print_token(token);
+  print_token("\"");
+}
+
+bool
+is_vscale_feat(char *mod)
```

Comment:
should take `const char *mod` instead. The `mod` variable is not modified by this function.